haystack/utils/openai_utils.py

```diff
@@ -75,12 +75,23 @@ def _openai_text_completion_tokenization_details(model_name: str):
     :param model_name: Name of the OpenAI model.
     """
     tokenizer_name = "gpt2"
-    if "davinci" in model_name:
-        max_tokens_limit = 4000
-        if USE_TIKTOKEN:
-            tokenizer_name = MODEL_TO_ENCODING.get(model_name, "p50k_base")
-    else:
-        max_tokens_limit = 2048
+    max_tokens_limit = 2049  # Based on the latest update.
```

Comment:
Thanks for checking this! Is there a link that has this update? If so could you provide that in this comment?

Reply:
All the information and links for all the changes are provided in the links below, and for this point, you can check this link in line 85 for the new token sizes update.